core/controllers/editor_test.py

```diff
@@ -1580,6 +1580,46 @@ def test_get_with_invalid_version_raises_error(self):
             params={'v': 'invalid_version'}, expected_status_int=404)
         self.logout()
 
+    def test_put_with_long_commit_message_raises_error(self):
+        # Create several users.
+        self.signup(self.COLLABORATOR_EMAIL, self.COLLABORATOR_USERNAME)
+
+        # Owner creates exploration.
+        self.login(self.OWNER_EMAIL)
+        exp_id = 'eid'
+        self.save_new_valid_exploration(
+            exp_id, self.owner_id, title='Title for rights handler test!',
+            category='My category')
+
+        exploration = exp_fetchers.get_exploration_by_id(exp_id)
+        exploration.add_states(['State A', 'State 2', 'State 3'])
+
+        csrf_token = self.get_new_csrf_token()
+
+        add_url = '%s/%s' % (feconf.EXPLORATION_DATA_PREFIX, exp_id)
```

Comment:
Do not start variables with verbs. Probably makes sense to just inline this one, like the other tests.

Reply:
Done. Also fixed this issue in unrelated tests.